examples/gno.land/r/demo/boards/z_4_filetest.gno

```diff
@@ -43,7 +43,7 @@ func main() {
 // > \- [@gnouser](/r/demo/users:gnouser), [2009-02-13 11:31pm (UTC)](/r/demo/boards:test_board/2/3) \[[reply](/r/demo/boards?help&__func=CreateReply&bid=1&threadid=2&postid=3&body.type=textarea)] \[[x](/r/demo/boards?help&__func=DeletePost&bid=1&threadid=2&postid=3)]
 //
 // > Second reply of the second post
-// > \- [@gnouser](/r/demo/users:gnouser), [2009-02-13 11:31pm (UTC)](/r/demo/boards:test_board/2/4) \[[reply](/r/demo/boards?help&__func=CreateReply&bid=1&threadid=2&postid=4&body.type=textarea)] \[[x](/r/demo/boards?help&__func=DeletePost&bid=1&threadid=2&postid=4)]
```

Comment:
println should be special-cased if it's just printing a string directly.

Reply:
What do you mean by special-cased?

Reply:
what I wrote afterwards in the review comment - it's kind of unexpected that println doesn't do what you want if the value is too long.